config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -344,7 +344,7 @@ presubmits:
         - --provider=gce
         - --runtime-config=api/all=true
         - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-alpha-features
-        - --test_args=--ginkgo.focus=\[Feature:(ServiceAccountIssuerDiscovery|StorageVersionHash|Audit|PodPreset|ExpandCSIVolumes|ExpandInUseVolumes|RunAsGroup|TTLAfterFinished|NodeLease|VolumeSnapshotDataSource|CSIInlineVolume|\s*CSIStorageCapacity)\]|Networking --ginkgo.skip=\[Feature:(SCTPConnectivity|Volumes|Networking-Performance)\]|IPv6|csi-hostpath-v0 --minStartupPods=8
+        - --test_args=--ginkgo.focus=\[Feature:(ServiceAccountIssuerDiscovery|StorageVersionHash|Audit|PodPreset|ExpandCSIVolumes|ExpandInUseVolumes|RunAsGroup|TTLAfterFinished|NodeLease|VolumeSnapshotDataSource|CSIInlineVolume|CSIStorageCapacity|GenericEphemeralVolume)\]|Networking --ginkgo.skip=\[Feature:(SCTPConnectivity|Volumes|Networking-Performance)\]|IPv6|csi-hostpath-v0 --minStartupPods=8
```

Comment:
Did you mean to drop the whitespace match? If so do we also want to drop it from https://github.com/kubernetes/test-infra/blob/ea5fa8f2188bfcbc3313861578770fdf6c2ca6b9/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L526? Did we want to add GenericEphemeralVolume testing to ci-kubernetes-e2e-gci-gce-alpha-features?

Reply:
Yes, yes, and yes. :grinning: 

I missed that there are two jobs which test alpha features. I thought there was only one.

I've updated also ci-kubernetes-e2e-gci-gce-alpha-features.